sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusSenderClient.java

```diff
@@ -71,6 +71,19 @@ public void send(ServiceBusMessage message) {
         asyncClient.send(message).block(tryTimeout);
     }
 
+    /**
+     * Sends an array of {@link ServiceBusMessage} to a Service Bus queue or topic using a batched approach.
+     * If the size of messages exceed the maximum size of a single batch, an exception will be triggered and the send
+     * will fail. By default, the message size is the max amount allowed on the link.
+     *
+     * @param messages Messages to be sent to Service Bus queue or topic.
+     *
```

Comment:
Specify the error that is thrown when the messages are too large for a single batch.
```java
@throws ...
```